docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -0,0 +1,57 @@
+= Parameters
+
+The parent key for all of the following parameters is `networkpolicy`.
+
+== `allowNamespaceLabels`
+
+[horizontal]
+type:: list of tuples
+default:: empty list
+
+A list of labels matching namespaces to allow traffic from.
+Each list item can contain several key value pairs.
+They result in an `AND` condition.
+Individual list iems will result in an `OR` condtion.
+
+[source,yaml]
+----
+allowNamespaceLabels:
+  - my-label-a: true
+    my-label-b: true
+  - my-label-c: true
+----
+
+In the above example, traffic will be allowed if a namespaces has the label `my-lable-a=true` AND `my-label-b=true`.
+Trafic will also be allowed if a namespce is labeled `my-label-c`.
+
+== `ignoredNamespaces`
+
+[horizontal]
+type:: list of strings
+default:: empty list
+
+A list of namespace names where no default NetworkPolicies will be created.
+All namespaces in this list will receive the label `espejo.syn.tools/no-network-policies=true`.
+Default NetworkPolicies will also be activly removed from thos namesapces.
+
+[NOTE]
+====
+The removal of the NetworkPolcies is done based on the namespace names.
+The label `espejo.syn.tools/no-network-policies` controlls only the creation of them.
```

Comment:
Wouldn't it make more sense to also use a label instead of using `matchNames`?
I think this would make it more consistent. The `matchNames` feature is really just a workaround for cases where one can't set labels on a namespace...

Reply:
Very good suggestion. This gives a generalized interface for pruning those default policies.